Gems/EditorPythonBindings/Code/Include/EditorPythonBindings/PythonUtility.h

```diff
@@ -127,5 +138,57 @@ namespace EditorPythonBindings
 
         //! Calls a BehaviorMethod with a tuple of arguments for member class level functions
         pybind11::object ClassMethod(AZ::BehaviorMethod* behaviorMethod, AZ::BehaviorObject self, pybind11::args args);
-    }
-}
+    } // namespace Call
+
+    namespace Text
+    {
+        class PythonBehaviorDescription
+        {
+        public:
+            //! Get Python type for behavior typeId.
+            AZStd::string_view FetchPythonTypeAndTraits(const AZ::TypeId& typeId, AZ::u32 traits);
+            AZStd::string FetchPythonTypeName(const AZ::BehaviorParameter& param);
+            AZStd::string FetchOutcomeType(const AZ::TypeId& typeId);
+
+            //! Creates a string containing bus events and documentation.
+            AZStd::string BusDefinition(const AZStd::string busName, const AZ::BehaviorEBus* behaviorEBus);
```

Comment:
nit: Should pass `busName` as a string_view instead.
```suggestion
            AZStd::string BusDefinition(const AZStd::string_view busName, const AZ::BehaviorEBus* behaviorEBus);
```